config/devstack.cfg

```diff
@@ -86,4 +86,23 @@ alias = roster
 number_of_shards = 5
 
 [problem-response]
-report_fields = ["username","problem_id","answer_id","problem","question","score","max_score","correct","answer","total_attempts","first_attempt_date","last_attempt_date"]
+report_fields = ["username","problem_id","answer_id","location","question","score","max_score","correct","answer","total_attempts","first_attempt_date","last_attempt_date"]
```

Comment:
Might be easier to visually diff this field later on if you broke each item out onto its own line.